### ISSUE TITLE
Allow aria-label to suppress template-accessibility-alt-text violation

### ISSUE THREAD

**Issue by ModestosV** [mgechev/codelyzer#976]
I am facing a use case where I do not want to specify an empty or non-empty alt text on an image element. However, I still want to provide an aria-label for assistive technologies.

Would it be possible for `template-accessibility-alt-text` to consider `aria-label` to be an acceptable alternative (either by default or as a rule option)?
